test/unit/settings_test.py

```diff
@@ -145,6 +145,26 @@ def test_eia860_years_overlap_eia860m_years(self: Self):
                 eia860m_year_months=[bad_eia860m_year_month],
             )
 
+    def test_eia860m_years_overlap_eia860m_years(self: Self):
+        """Test validation error is raised when eia860m years overlap."""
+        max_eia860_year = max(Eia860Settings().years)
+        acceptable_eia860m_year = max_eia860_year + 1
+        bad_eia860m_year_months = [
+            f"{acceptable_eia860m_year}-01",
+            f"{acceptable_eia860m_year}-05",
+        ]
+        with pytest.raises(ValidationError):
+            _ = Eia860Settings(
+                eia860m=True,
+                years=[max_eia860_year],
+                eia860m_year_months=bad_eia860m_year_months,
+            )
+
+    def test_eia860m(self: Self):
+        """Test creation of eia860m_year_month values when eia860m is True."""
+        eia860_settings = Eia860Settings()
```

Comment:
I think it's more robust and readable to explicitly set `eia860m=True` in the arguments here, rather than relying implicitly on the default value.